source/includes/changelog/_19-09-0.md.erb

```diff
@@ -1,3 +1,20 @@
 ## Changes in 19.9.0
 
-### No changes
\ No newline at end of file
+### Added
+
+- Introduced version 3 of [Environment Config API](#environment-config)
+- Introduced version 6 of [Agents API](#agents)
+
+### Removed
+
+- <%= removed_api_message({api_name: 'Environment Config', old_api_version: 2, new_api_version: 3, api_section: 'environment-config'}) %>
+- <%= removed_api_message({api_name: 'Agents', old_api_version: 4, new_api_version: 6, api_section: 'agents'}) %>
```

Comment:
Should we add a note for removal without deprecation?

Reply:
Makes sense. Should we link it to the agents PR?

Reply:
We can.

Reply:
Updated the changelog.